packages/ci/test/commands/ci/run.test.ts

```diff
@@ -207,6 +202,8 @@ describe('ci:run', () => {
       .stub(git, 'createArchive', gitFake.createArchive)
       .stub(fs, 'stat', fsFake.stat)
       .stub(fs, 'createReadStream', fsFake.createReadStream)
+      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+      // @ts-ignore-next-line
```

Comment:
I think I remember you telling me why this was necessary, but I can't remember. Would you mind adding a comment explaining the `@ts-ignore-next-line`? Just to document.